AppCenterDataStorage/AppCenterDataStorageTests/MSDataStoreTests.m

```diff
@@ -58,12 +58,14 @@ @implementation MSDataStoreTests
 static NSString *const kMSTestAppSecret = @"TestAppSecret";
 static NSString *const kMSCosmosDbHttpCodeKey = @"com.Microsoft.AppCenter.HttpCodeKey";
 static NSString *const kMSTokenTest = @"token";
-static NSString *const kMSPartitionTest = @"partition";
+static NSString *const kMSPartitionTest = @"user";
 static NSString *const kMSDbAccountTest = @"dbAccount";
+static NSString *const kMSAccountId = @"ceb61029-d032-4e7a-be03-2614cfe2a564";
 static NSString *const kMSDbNameTest = @"dbName";
 static NSString *const kMSDbCollectionNameTest = @"dbCollectionName";
 static NSString *const kMSStatusTest = @"status";
-static NSString *const kMSExpiresOnTest = @"20191212";
+static NSString *const kMSExpiresOnTest = @"2999-09-19T11:11:11.111Z";
+;
 static NSString *const kMSDocumentIdTest = @"documentId";
 
 - (void)setUp {
```

Comment:
Since Datastore is now using a dispatch queue, we'd like to make sure the queue doesn't execute any other code while the test is done. There is an example there: https://github.com/Microsoft/AppCenter-SDK-Apple/blob/a3dee08381714c300d501f8de61531cf58024d05/AppCenter/AppCenterTests/MSChannelGroupDefaultTests.m#L54